src/test/java/org/apache/sling/resourceresolver/impl/mapping/EtcMappingMapEntriesTest.java

```diff
@@ -16,12 +16,40 @@
  */
 package org.apache.sling.resourceresolver.impl.mapping;
```

Comment:
There are a lot of changes in this test and I can't see whether these are just improvements or tests need to be adapted?

Reply:
As far as I can tell I did not change anything here beside that I might have removed commented out code.

Reply:
There are some additional tests cases to deal with types and defaults but the rest did change little.